src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/Configuration/AzureADSettingsConfiguration.cs

```diff
@@ -0,0 +1,61 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using OrchardCore.Environment.Shell.Models;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Microsoft.Authentication.Services;
+using OrchardCore.Microsoft.Authentication.Settings;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.Extensions.Logging;
+
+namespace OrchardCore.Microsoft.Authentication.Configuration;
+
+public class AzureADSettingsConfiguration : IConfigureOptions<AzureADSettings>
+{
+    private readonly IAzureADService _azureADService;
+    private readonly ShellSettings _shellSettings;
+    private readonly ILogger _logger;
+
+    public AzureADSettingsConfiguration(
+        IAzureADService azureADService,
+        ShellSettings shellSettings,
+        ILogger<AzureADOptionsConfiguration> logger)
```

Comment:
=> `ILogger<AzureADSettingsConfiguration>`.

Review the `ILogger<>` types in all other places ;)

Reply:
Oops, copy paste issue ;)

Reply:
Oops, copy paste issue ;)